ai/poc/model_builder.py

```diff
@@ -0,0 +1,98 @@
+from keras.layers import Input, Dense, Flatten
+from keras.models import Model
+import numpy as np
+import json
+
+json_data = json.load(
+    open('./trainning_data/trainning_data.json'))
+
+data = []
+
+
+def clean_data(data):
+    for match in data:
+        if match[0] == 0 or match[1] == 0:
+            data.remove(match)
+            continue
+        if match[2] == 0 or match[3] == 0:
+            data.remove(match)
+            continue
+        if any(len(x) != 15 for x in match[0]) or any(len(x) != 15 for x in match[1]):
```

Comment:
What is tested here ?

Reply:
That file contains the code of an old poc and I'll move it to a more dedicated place on a next PR
but the line 20 check that we have all the units stats (sorry for the magic value 15, I'll patch it on this other branch)
and the lines above are basically mistakes :/ don't bother

Reply:
So will those be deleted in this PR ? If not add an issue because we might forget

Reply:
Yup those are patched, the clean is ready on an other branch but I'm waiting the validation here to rebase the changes